assets/js/vue-apps/past-grants.js

```diff
@@ -59,6 +59,11 @@ function init() {
             }
         },
         mounted() {
+            // Enable inputs (they're disabled by default to avoid double inputs for non-JS users)
```

Comment:
This really needed to be here otherwise for non-JS users the form has two `q` inputs which both get sent (and the API fails)

Reply:
Ah, thought I'd resolved this by virtue of the fact Vue now replaces the markup for all the inputs, but forgot about the search form! Good catch